src/lib.rs

```diff
@@ -506,6 +535,49 @@ impl Logger {
         self
     }
 
+    /// Sets the output for a level.
+    ///
+    /// The output is either `stderr` or `stdout`. The default is for ERROR and WARN to be written
+    /// to `stdout` and INFO, DEBUG, and TRACE to stdout.
```

Comment:
typo here, both stdout

Reply:
Fixed in 2377bb8d6d449cef5e15b27b9395c2d964209047